src/components/MoneyRequestConfirmationList.js

```diff
@@ -663,11 +663,15 @@ function MoneyRequestConfirmationList(props) {
                 title={props.iouComment}
                 description={translate('common.description')}
                 onPress={() => {
-                    if (props.isEditingSplitBill) {
-                        Navigation.navigate(ROUTES.EDIT_SPLIT_BILL.getRoute(props.reportID, props.reportActionID, CONST.EDIT_REQUEST_FIELD.DESCRIPTION));
-                        return;
-                    }
-                    Navigation.navigate(ROUTES.MONEY_REQUEST_DESCRIPTION.getRoute(props.iouType, props.reportID));
+                    Navigation.navigate(
+                        ROUTES.MONEY_REQUEST_STEP_DESCRIPTION.getRoute(
+                            CONST.IOU.ACTION.EDIT,
```

Comment:
Why are we assuming this is an edit? We should use the `action` param (same for `MoneyTemporaryForRefactorRequestConfirmationList`)

Reply:
@s77rt  Currently, we only use MoneyRequestConfirmationList in SplitBillDetailsPage (to **edit**) and SplitBillDetailsPage doesn't have an action param in the route because it is always in be EDIT flow. One more thing, we use `MoneyTemporaryForRefactorRequestConfirmationList` only for creating flow. Note that, Confirmation Page only exist in create flow

Reply:
Ah, It seems `MoneyRequestConfirmationList` also be removed and in SplitBillDetailsPage 
 we also use `MoneyTemporaryForRefactorRequestConfirmationList` instead. So that `MoneyTemporaryForRefactorRequestConfirmationList` also be used for edit split bill. So that I think we need to wait until [this issue](https://github.com/Expensify/App/issues/34615) is done. Then we can get `action` params form route to update here

Reply:
Is there an ETA for that issue?

Reply:
@s77rt for the option 1 in [here](https://github.com/Expensify/App/pull/35137#discussion_r1475730296). 

we can handle it in `Remove MoneyRequestConfirmationList.js and copy into MoneeRequestConfirmationList.js`. This issue also be created soon as mentioned [here](https://github.com/Expensify/App/issues/29107#issuecomment-1924216766)

Reply:
Let's go with option 1 then